stable/unifi/OWNERS

```diff
@@ -1,4 +1,6 @@
 approvers:
 - billimek
+- mcronce
```

Comment:
OWNERS must also be listed as maintainers in `Chart.yaml`. Should we invite @mcronce as collaborator? He'll be expected to actively contribute to the chart, review PRs, triage issues, etc.

Reply:
@mcronce is now added as a maintainer in the `Chart.yml`, and it would be great if they could be added as a collaborator as well, thank you!

Reply:
done